Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -589,8 +589,8 @@ public IEnumerable<EntityUid> GetColliding(EntityUid uid, PhysicsComponent? phys
             if (otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.GlassLayer || otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.GlassAirlockLayer || otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.TableLayer)
                 continue;
 
-            //If the colliding entity is a slippable item ignore it by the airlock
-            if (otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.SlipLayer && otherPhysics.Comp.CollisionMask == (int) CollisionGroup.ItemMask)
+            //If the colliding entity is a non-lowImpassible, ignore it by the airlock
+            if ((otherPhysics.Comp.CollisionMask & (int)CollisionGroup.LowImpassable) == 0)
```

Comment:
```suggestion
            // Ignore low-passable entities.
```

imo the double negative of `non-LowImpassible` combined with the `if $condition, then result` grammar makes this harder to semantically parse as a reader than the code itself, rendering the comment unhelpful.

Reply:
Very true, thanks!